wrphttp/requestResponse.go

```diff
@@ -71,7 +92,7 @@ type ResponseWriter interface {
 	// WriteWRP writes a WRP message to the underlying response.  The format used is determined
 	// by the configuration of the underlying implementation.  This method is idempotent, and returns
 	// an error if called multiple times for the same instance.
-	WriteWRP(interface{}) (int, error)
+	WriteWRP(e *Entity) (int, error)
```

Comment:
This is the right idea.  But we also want a `WriteXXX(wrp.Format, []byte)` method. That allows for proxying and pass through, where a caller never deserialized the `[]byte` into a `wrp.Message`.